files/es/web/html/element/title/index.html

```diff
@@ -8,48 +8,123 @@
 translation_of: Web/HTML/Element/title
 original_slug: Web/HTML/Elemento/title
```

Comment:
necesita la llave para que compat funcioné

```suggestion
original_slug: Web/HTML/Elemento/title
browser-compat: html.elements.title
```